tests/pytests/unit/grains/test_core.py

```diff
@@ -1154,6 +1154,36 @@ def test_almalinux_8_os_grains():
     }
     _run_os_grains_tests(_os_release_data, {}, expectation)
 
+@pytest.mark.skip_unless_on_linux
+def test_virtuozzo_7_os_grains():
+    """
+    Test if OS grains are parsed correctly in Virtuozzo 7
+    """
+    _os_release_data = {
+        "NAME": "Virtuozzo",
+        "ID": "virtuozzo",
+        "PRETTY_NAME": "Virtuozzo release 7.5.4",
+        "VERSION": "7.5.4",
+        "ID_LIKE": "rhel fedora",
+        "VERSION_ID": "7",
+        "ANSI_COLOR": "0;31",
+        "CPE_NAME": "cpe:/o:virtuozzoproject:vz:7",
+        "HOME_URL": "http://www.virtuozzo.com",
+        "BUG_REPORT_URL": "https://bugs.openvz.org",
+    }
+    expectation = {
+        "os": "Virtuozzo",
+        "os_family": "RedHat",
+        "osfullname": "Virtuozzo",
+        "oscodename": "Virtuozzo release 7.5.4",
+        "osrelease": "7",
+        "osrelease_info": (7),
```

Comment:
```suggestion
        "osrelease_info": (7),
```
You might be expecting a tuple here?!

```suggestion
        "osrelease_info": (7, 5, 4),
```
Or
```suggestion
        "osrelease_info": (7, 5),
```

Reply:
Thank you. I believe with VERSION_ID=7 it should return (7,) so I changed it to
`"osrelease_info": (7,),`